guides/decoding-blocks-transactions/getBlockById.js

```diff
@@ -0,0 +1,32 @@
+const { apiClient } = require('@liskhq/lisk-client');
+let clientCache;
+const nodeAPI = 'ws://localhost:8080/ws';
```

Comment:
I think `nodeAPIURL` is better(not sure about it should be URL or Url) because nodeAPI looks like an object name.